cmake/IncludeWhatYouUse.cmake

```diff
@@ -0,0 +1,11 @@
+if(ENABLE_IncludeWhatYouUse)
```

Comment:
Suggest to place this file in cmake/nebula

Reply:
should place in cmake. This is a general cmake tool

Reply:
ok I change it to CAPITAL